lms/djangoapps/ccx/views.py

```diff
@@ -410,7 +410,7 @@ def ccx_invite(request, course, ccx=None):
     action = request.POST.get('enrollment-button')
     identifiers_raw = request.POST.get('student-ids')
     identifiers = _split_input_list(identifiers_raw)
-    auto_enroll = True if 'auto-enroll' in request.POST else False
+    auto_enroll = True
```

Comment:
yes i think so, but we are ignoring value. so that auto enroll always true.
By the way FYI checkbox is part of a form.